packages/grafana-data/src/types/dataFrame.ts

```diff
@@ -101,6 +101,8 @@ export interface FieldConfig<TOptions = any> {
 
   actions?: Action[];
 
+  oneClickMode?: OneClickMode;
```

Comment:
Having this be a new global (all field) level config option feels wrong somehow, is there a doc or issue describing the problem we are trying to solve? I doubt a new global field level option is warranted